docs/.project/RELEASE.md

```diff
@@ -22,45 +24,55 @@ For patch releases turn to [this documentation](./PATCH_RELEASE.md).
+    * [ ] wait for [Release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
 * [ ] execute [integration test](https://github.com/camunda/camunda-modeler/blob/master/docs/.project/INTEGRATION_TEST.md) on [released artifacts](https://github.com/camunda/camunda-modeler/releases)
     * [ ] Works on Linux
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] notify QA about the release so they can test it
-* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
-   * [ ] create a draft post on Wordpress
-   * [ ] send the draft to `Christopher Rogers` to coordinate the publishing
-* [ ] (if the Modeler UI appearance changed) update Camunda Modeler screenshots (and potentially text content) in the docs using [automation scripts](https://github.com/camunda/camunda-docs-modeler-screenshots)
 
 _To be done after integration test was successful or if not issues were fixed._
 
 * [ ] create release (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema)
-    * [ ] wait for [GitHub Actions](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to build the executables
-* [ ] send all information required to compile the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email) to the [`@product-release-presentation-dri` role](https://confluence.camunda.com/pages/viewpage.action?spaceKey=camBPM&title=Release+Presentation+Organization)
+    * [ ] wait for [Release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
+
+_To be done once the release is built._
+
+* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
+   * [ ] create a draft post on Wordpress
+   * [ ] send the draft to `Christopher Rogers` to coordinate the publishing
+* [ ] (if the Modeler UI appearance changed) update Camunda Modeler screenshots (and potentially text content) in the docs using [automation scripts](https://github.com/camunda/camunda-docs-modeler-screenshots)
+* [ ] send all information required to compile the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email) to the [`@product-release-presentation-dri` role](https://confluence.camunda.com/pages/viewpage.action?spaceKey=HAN&title=Release+Presentation+Process#ReleasePresentationProcess-OrganisingtheReleasePresentation)
 * [ ] reach out to `David Paradis` so that he updates download version of [Camunda Modeler](https://camunda.com/de/download/modeler/) on release day
 
 _To be done on release day to announce the release and making it publically available._
```

Comment:
Technically with these steps we do not announce it publically (that would be marketing publishing the blog post right)?

```suggestion
_To be done on release day._
```

Reply:
See my next comments, maybe this section is obsolete.